.gitignore

```diff
@@ -13,6 +13,7 @@ benchmark/dist
 .sandbox
 packages/cli/generators/datasource/connectors.json
 docs/site/readmes
+**/tsconfig.build.tsbuildinfo
```

Comment:
Let's use `dist/.tsbuildinfo` by configuring tsconfig compilerOptions with:
```json
{
  "tsBuildInfoFile": ".tsbuildinfo"
}
```

This way, no changes is needed in `clean` script.